src/AriaComponent.js

```diff
@@ -101,20 +119,74 @@ export default class AriaComponent {
     return this.stringDescription;
   }
 
+  /**
+   * Dispatch event.
+   *
+   * @param  {string} name   The event name.
+   * @param  {object} detail The event detail object.
+   */
+  dispatch(name, detail) {
+    if (! this._suppressDispatch.includes(name)) {
+      const event = new CustomEvent(
+        name,
+        {
+          bubbles: true,
+          composed: true,
+          detail,
+        }
+      );
+
+      this.element.dispatchEvent(event);
+    }
```

Comment:
Idea to suppress _bubbling_ rather than the event. The event itself may prove useful in certain circumstances.

```suggestion
    const event = new CustomEvent(
      name,
      {
        bubbles: (! this._suppressDispatch.includes(name)),
        composed: true,
        detail,
      }
    );

    this.element.dispatchEvent(event);
```

Reply:
🙅